src/main/java/org/tikv/common/KVClient.java

```diff
@@ -141,7 +141,7 @@ public ByteString get(ByteString key, long version) throws GrpcException {
           () -> doSendBatchGetInBatchesWithRetry(singleBatchBackOffer, batch, version));
     }
 
-    return getKvPairs(completionService, batches);
+    return getKvPairs(completionService, batches, BackOffer.BATCH_GET_MAX_BACKOFF);
```

Comment:
it's not related to the pr title.

Reply:
It is added because the corresponding API in `ClientUtils` has changed.

Reply:
... which corresponds to the new-added `ClientUtils.getTasks()` API